src/duplo_resource/s3.py

```diff
@@ -0,0 +1,24 @@
+from duplocloud.client import DuploClient
+from duplocloud.resource import DuploTenantResourceV3
+from duplocloud.commander import Command, Resource
+import duplocloud.args as args
+
+
+@Resource("s3")
+class DuploS3(DuploTenantResourceV3):
+  def __init__(self, duplo: DuploClient):
+    super().__init__(duplo, "aws/s3bucket")
+
+  @Command()  
+  def create(self, 
+             body: args.BODY):
+    """Create a job."""
+    name = self.name_from_body(body)
+    print("Running create")
```

Comment:
don't use print. 
If you really want to log something, 
do this instead: 
`self.duplo.logger.debug("Running create s3")`

Print will always go to standard out and then be included with the output of the cli. 
So something like this would include your log: 
`export THING="$(duploctl s3 create -f file.yaml)"`

I'm also pretty sure you don't need to override this method?